integration/rest/src/main/java/it/gov/pagopa/rtd/transaction_filter/connector/HpanRestClientImpl.java

```diff
@@ -249,11 +260,17 @@ public Void uploadFile(File fileToUpload, String sas, String authorizedContainer
 
     HpanRestConnectorConfig config = context.getBean(HpanRestConnectorConfig.class);
     SSLContext sslContext = config.getSSLContext();
+    HttpHost proxy = new HttpHost(proxyHost, proxyPort);
```

Comment:
Can new HttpHost raise an exception if proxyHost and proxyPort are nulls? E.g. when proxy is disabled?

Reply:
If the parameters are the default ones there will be no exception raised. Anyway it's a good point and it can be improved, thank you!